src/d3a/models/strategy/area_agents/one_sided_engine.py

```diff
@@ -125,9 +122,10 @@ def propagate_offer(self, current_tick):
             if self.owner.name == offer.seller:
                 continue
 
-            forwarded_offer = self._forward_offer(offer, offer_id)
+            forwarded_offer = self._forward_offer(offer)
             if forwarded_offer:
-                self.owner.log.debug("Offering %s", forwarded_offer)
+                self.owner.log.debug(f"Offering on {self.markets.source.name} "
```

Comment:
```suggestion
                self.owner.log.debug(f"Forwarded Offer on MARKET: {self.markets.source.name} "
```

Reply:
Thanks for this suggestion! I definitely agree to change this (confused this when debugging).